_posts/2016-10-05-gazette-launch.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+author: Lenina Rassool
+title: "Get access to your government gazette online - for free"
+excerpt: |
+    A new website, opengazettes.org.za, is making official government gazettes available online and searchable, so you no longer have to sift through or download heaps of publications to source important, public information.
+image: /images/blog/gazette-launch.png
+---
+
+<img src="/images/blog/gazette-launch.png" style="width:100%;">
+
+A new website, [opengazettes.org.za](http://opengazettes.org.za/), is making official government gazettes available online and searchable, so you no longer have to sift through or download heaps of publications to source legal notices, calls for comments on national and provincial matters and updates on civic issues. 
+ 
+The project is a collaboration between [Code for South Africa](http://code4sa.org), a civic-technology organisation that promotes informed decision-making, the [Southern African Legal Information Institute](http://www.saflii.org) (SAFLII), [The Indigo Trust](http://www.indigotrust.org.uk) and the [African Networks of Centers for Investigative Reporting](https://www.investigativecenters.org), with support from [Code for Africa](https://codeforafrica.org). The initiative aims to promote citizen access to government information and increase civic participation in government processes.
+ 
+According to Parliament’s website, "Government uses [the Gazette] to publish acts and bills, regulations and notices in terms of acts, changes of names, company registrations and deregistrations, financial statements, land restitution notices, liquor license applications and transport permits. Board and legal notices are also published in the Gazette; these cover insolvencies, liquidation and estate notices."
+
+In a nutshell, this includes everything the government is doing that citizens should know about. 
+ 
+While government gazettes have always been made available somewhere, access to the information is not as easy or straightforward as it sounds, especially for regular citizens. This is especially worrying when the onus is on the public to "keep up to date with all legislation published in the Government Gazette, and ignorance is not permitted as a defense for violating a law or regulation", according to the [Official Publications Depository Manual](http://www.nlsa.ac.za/downloads/OPD%20Manual%20Rev%20Ed%20Aug%202007.pdf), which hosts guidelines for setting up depositories for official publications across South Africa.
```

Comment:
Guus and I agree that 'access to the information is not as easy or straightforward as it sounds' implies something that hasn't necessarily been indicated by the preceding text. It sounds like the process is complicated at best--a problem that Open Gazettes aims to solve. Maybe it would be better to say "accessing them is not as straightforward as it should be?"